Cargo.toml

```diff
@@ -15,12 +15,12 @@ members = [
 lto = "thin"
 
 [workspace.dependencies]
-gateway-api = "0.15"
+gateway-api = "0.16"
 http = "1"
 hyper = "1"
-k8s-openapi = { version = "0.24", features = ["v1_32"] }
-kube = { version = "0.99", default-features = false }
-kubert = { version = "0.24", default-features = false }
+k8s-openapi = { version = "0.25", features = ["v1_33"] }
+kube = { version = ">=1.1.0,<1.2.0", default-features = false }
```

Comment:
this is connected to https://github.com/kube-rs/kube/discussions/1789, right?

could we leave a comment here to help connect those dots for future readers?

Reply:
it's not necessary